### DISCUSSION TITLE
List of Requested Features

### DISCUSSION THREAD

**Discussion by malliaridis** [malliaridis/obssd#49] [General]
## Welcome to the Feature requests page

We are collecting here all the feature requests of the users before introducing them in the placeholder tab of the OBSSD Premium app.

### Current Ideas

- Live Stream to watch the stream from inside the application (main focus group are streamers)
- Twitch Chat that allows reading and chatting with viewers, eventually including moderation options (main focus group are streamers)
- AFK Features that allows an easy switch back and forth to a predefined AFK scene (main focus group are streamers)

### You got an Idea or like one of the above?

If you wish something else or like any of the above ideas, feel free to write a comment, leave a reaction to any of the ideas you like or write me a message via email (especially if you don't want to create a Github-Account solely for this).

### When will we see the new feature(s)?

This part of the app will be implemented once there is a "clear user preference". This can be sooner or later, depending on the prioritization and user feedback.

_Note: Prefer to use english here and english german or greek in the email so that your request can be taken into account._

**Comment by skueee** [comment]
A option mode, for toogle studio mode, stream and more
